lang/po/Firmware_sk.po

```diff
@@ -2585,6 +2585,12 @@ msgstr "Nie je zavedený žiaden filament."
 msgid "Z calibration recommended. Run it now?"
 msgstr "Odporúča sa kalibrácia Z. Spustiť ju teraz?"
 
+#. MSG_CHECK_SHEET_TYPE c=20 r=3
+#: ../../Firmware/Marlin_main.cpp:5844 ../../Firmware/messages.cpp:163
+#: ../../Firmware/util.cpp:454 ../../Firmware/util.cpp:455
+msgid "Check selected steel sheet."
+msgstr "Skontrolujte vybraný platňa."
```

Comment:
Skontrolujte vybranú platňu

Reply:
Thanks, will wait for other reviews and commit later